plugins/experimental/ts_lua/ts_lua.c

```diff
@@ -116,6 +116,69 @@ TSRemapDeleteInstance(void *ih)
   return;
 }
 
+void
+TSRemapOSResponse(void *ih, TSHttpTxn rh, int os_response_type)
```

Comment:
This function is practically identical to ``TSRemapdoRemap()``. Factor into a common subroutine.